src/csharp/Grpc.Tools/build/_grpc/_Grpc.Tools.targets

```diff
@@ -32,6 +32,13 @@
   </Target>
 
   <Target Name="_gRPC_PrepareCompileOptions" AfterTargets="Protobuf_PrepareCompileOptions">
+
+    <PropertyGroup>
+      <!-- Enable nullabe reference types if enabled in the project and not been overridden
+           by setting gRPC_NullableReferenceTypes -->
+      <gRPC_NullableReferenceTypes Condition=" '$(gRPC_NullableReferenceTypes)' == '' and '$(Nullable)' == 'Enable' ">Enable</gRPC_NullableReferenceTypes>
```

Comment:
I think besides being able to set the NRT handling with gRPC_NullableReferenceTypes and Protobuf_NullableReferenceTypes separately (which seems useful), we'd also need a property that enables/disables both at the same time (since likely that's what users will want if they care about NRT and if both protoc and grpc_csharp_plugin support NRT option in the future).

Or should we rely on most users simply having `Nullable == Enable` and therefore the NRT codegen being used by default (and the gRPC_NullableReferenceTypes and Protobuf_NullableReferenceTypes will only be there as an option to disable it if needed?).